fabfile.py

```diff
@@ -794,39 +828,92 @@ def _install_builddeps():
             run('brew install gcc isl')
     else:
         sudo('apt-get install -y build-essential python-dev pandoc')
-        sudo('apt-get install -y automake bison flex gperf  libxml2-dev libssl-dev libreadline-dev gawk')
-        sudo('apt-get install -y graphviz libgraphviz-dev pkg-config')
-        sudo('apt-get install -y liblapack-dev libatlas-dev libblas-dev gfortran')
-        print "We are still trying to get some requirements right for linux,"\
-            "See http://www.scipy.org/scipylib/building/linux.html for details."
-
-        # Runtime requirements (even in develop)
-        sudo('apt-get install -y redis-server memcached unixodbc-dev')
-    execute(install_database)
-    execute(update_python_package_builddeps)
+        sudo('apt-get install -y automake bison flex gperf gawk')
```

Comment:
Aka...I understand they are needed to proceed with Assembl install, no clue as to how they were identified.